src/libserver/cfg_rcl.c

```diff
@@ -1784,9 +1784,9 @@ rspamd_rcl_config_init (struct rspamd_config *cfg, GHashTable *skip_sections)
 				0,
 				"Add all symbols only once per message");
 		rspamd_rcl_add_default_handler (sub,
```

Comment:
Uh-oh, we need to add a way to support both options here I'm afraid...

Reply:
I'm tempted to drop this commit since that's a bit too complicated for what I'm doing. Unless you can easily point me to some logic I can copy.

Reply:
Yes, let's drop it and I will make a follow up change.

Reply:
Pushed to https://github.com/jsoref/rspamd/tree/spelling-attachment